roles/nf-core/templates/site.config.j2

```diff
@@ -1,5 +1,4 @@
 process {
```

Comment:
This should be in an if with (methylseq or ampliseq) as it will give you an empty process for other pipelines which will cause an error when you run them.